docs/tutorials/02_ui/viz_card.py

```diff
@@ -0,0 +1,59 @@
+# -*- coding: utf-8 -*-
+"""
+===============
+Buttons & Text
+===============
+
+This example shows how to use the UI API. We will demonstrate how to create
+panel having multiple cards on them.
+
+First, some imports.
+"""
+from numpy.core.fromnumeric import size
+from fury import ui, window
+from fury.data import read_viz_icons, fetch_viz_icons
+
```

Comment:
Remove the `numpy.core.fromnumeric...` import as its not being used in your script.